pkg/segment/aggregations/segaggs.go

```diff
@@ -121,6 +121,12 @@ func PostQueryBucketCleaning(nodeResult *structs.NodeResult, post *structs.Query
 	return nodeResult
 }
 
+/*
+* TODO: The processing logic for SPL commands that need to be implemented is outlined below. These commands may need to be implemented elsewhere.
+ 1. Stats cmd:
+    1.1 stats options: dedup_splitvals, allnum, partitions, delim
+    1.2 stats functions: estdc, estdc_error, exactperc99, perc66.6, median, stdev, stdevp, sumsq, upperperc6.6, var, varp, first, last, list, earliest, earliest_time, latest, latest_time, rate
+*/
 func performAggOnResult(nodeResult *structs.NodeResult, agg *structs.QueryAggregators, recs map[string]map[string]interface{},
```

Comment:
I added the error handling logic after the parser finished parsing, because there might be some issues with `nodeResult.ErrList`. Even when I arbitrarily added an error to that list, the frontend did not display anything, and I could not find where `ErrList` is returned. I have created a GitHub issue for that. https://github.com/siglens/siglens/issues/1101